sys/net/gnrc/network_layer/ipv6/nib/nib.c

```diff
@@ -259,38 +314,37 @@ void gnrc_ipv6_nib_handle_timer_event(void *ctx, uint16_t type)
             _handle_snd_na(ctx);
             break;
         case GNRC_IPV6_NIB_SEARCH_RTR:
-            /* TODO */
-            break;
-        case GNRC_IPV6_NIB_RECONFIRM_RTR:
-            /* TODO */
+            _handle_search_rtr(ctx);
             break;
 #if GNRC_IPV6_NIB_CONF_ROUTER
         case GNRC_IPV6_NIB_REPLY_RS:
-            /* TODO */
+            _handle_reply_rs(ctx);
             break;
         case GNRC_IPV6_NIB_SND_MC_RA:
-            /* TODO */
+            _handle_snd_mc_ra(ctx);
             break;
 #endif  /* GNRC_IPV6_NIB_CONF_ROUTER */
-#if GNRC_IPV6_NIB_CONF_6LN
+#if GNRC_IPV6_NIB_CONF_6LR
         case GNRC_IPV6_NIB_ADDR_REG_TIMEOUT:
-            /* TODO */
-            break;
-        case GNRC_IPV6_NIB_6LO_CTX_TIMEOUT:
-            /* TODO */
+            _nib_nc_remove(ctx);
             break;
 #endif  /* GNRC_IPV6_NIB_CONF_6LN */
 #if GNRC_IPV6_NIB_CONF_MULTIHOP_P6C
         case GNRC_IPV6_NIB_ABR_TIMEOUT:
-            /* TODO */
+            _nib_abr_remove(&((_nib_abr_entry_t *)ctx)->addr);
             break;
 #endif  /* GNRC_IPV6_NIB_CONF_MULTIHOP_P6C */
         case GNRC_IPV6_NIB_PFX_TIMEOUT:
-            /* TODO */
+            _handle_pfx_timeout(ctx);
             break;
         case GNRC_IPV6_NIB_RTR_TIMEOUT:
-            /* TODO */
+            _handle_rtr_timeout(ctx);
             break;
+#if GNRC_IPV6_NIB_CONF_6LN
+        case GNRC_IPV6_NIB_REREG_ADDRESS:
+            _handle_rereg_address(ctx);
+            break;
+#endif
```

Comment:
This `#endif` is sad because it doesn't have a comment (the rest of the `#endif`s here have one).

There are probably more sad `#endif`s scattered throughout the code, but I noticed this one because it is the only one in a sequence without the comment.